nodes/test/test_pinecone_record_updates.py

```
@@ -0,0 +1,309 @@
+    index = client.Index(_INDEX_NAME)
+    if vectors:
+        index.upsert(vectors)
+
+    store = Store.__new__(Store)
+    store.collection = _INDEX_NAME
+    store.client = client
+    store.doesCollectionExist = lambda *_a, **_kw: True
+    return store, index
+
+
+def _get_records(index: MockIndex) -> dict[str, dict]:
```

Ruff (0.15.6)

[warning] Unused function argument: `index`

(ARG001)